test/test_apps/5.0/Appraisals

```diff
@@ -0,0 +1,40 @@
+case(RUBY_VERSION)
+when "2.2.5" then
+
+  appraise "capybara-2.2-ruby#{RUBY_VERSION}" do
```

Comment:
So many capybaras...are they all necessary?
On the other hand, there are newer versions as well.